aptos-move/move-examples/mint_nft_v2/1-Create-NFT/sources/create_nft.move

```diff
@@ -0,0 +1,187 @@
+        let description = string::utf8(b"Description");
+        let collection_uri = string::utf8(b"Collection uri");
+        let token_name = string::utf8(b"Token name");
+        let token_uri = string::utf8(b"Token uri");
+        let maximum_supply = 1000;
+
+        aptos_token::create_collection(
+            source_account,
+            description,
+            maximum_supply,
+            collection_name,
+            collection_uri,
+            false, // mutable_description
+            false, // mutable_royalty
+            false, // mutable_uri
+            false, // mutable_token_description
+            false, // mutable_token_name
+            true, // mutable_token_properties
+            false, // mutable_token_uri
+            false, // tokens_burnable_by_creator
+            false, // tokens_freezable_by_creator
+            5, // royalty_numerator
+            100, // royalty_denominator
+        );
+
+        let source_address = signer::address_of(source_account);
+        let collection_object_address = collection::create_collection_address(&source_address, &collection_name);
+
+        move_to(source_account, ModuleData {
+            collection_name,
+            creator: source_address,
+            token_description: string::utf8(b""),
+            token_name,
+            token_uri,
+            property_keys: vector<String>[string::utf8(b"given_to")],
+            property_types: vector<String>[ string::utf8(b"address") ],
+            property_values: vector<vector<u8>>[bcs::to_bytes(&source_address)],
+            collection_object_address,
+        });
+
+    }
+
+    /// Mint an NFT to the receiver. Note that we don't need the receiver to sign to receive a token/object
+    /// with token v2- you only need to pass the `receiver_address` to the entry function.
+    public entry fun delayed_mint_event_ticket(module_owner: &signer, receiver_address: address) acquires ModuleData {
+        let module_owner_address = signer::address_of(module_owner);
+        assert!(module_owner_address == @mint_nft_v2, error::permission_denied(ENOT_AUTHORIZED));
+
+        let module_data = borrow_global_mut<ModuleData>(@mint_nft_v2);
+
+        // mint token to the receiver
+        let token_creation_num = account::get_guid_next_creation_num(module_owner_address);
+        aptos_token::mint(
+            module_owner,
+            module_data.collection_name,
+            module_data.token_description,
+            module_data.token_name,
+            module_data.token_uri,
+            module_data.property_keys,
+            module_data.property_types,
+            module_data.property_values,
+        );
+        let token_object = object::address_to_object<AptosToken>(object::create_guid_object_address(module_owner_address, token_creation_num));
+        object::transfer(module_owner, token_object, receiver_address);
+
+        // update "given_to" to the value of the new receiver.
```

Comment:
Is this still useful in token v2?

Reply:
Showing who the owner is in the property map? No, not really. I'll change it to something random like "Date Minted" just to show how to use the property map still